modules/apps/headless/headless-delivery/headless-delivery-test/src/testFunctional/macros/DocumentFolderAPI.macro

```diff
@@ -68,4 +170,51 @@ definition {
 		return ${response};
 	}
 
+	macro getDocumentFoldersWithDifferentPameter {
+		var curl = DocumentFolderAPI._curlDocumentFolders(
+			groupName = ${groupName},
+			parameter = ${parameter},
+			parameterValue = ${parameterValue});
+
+		var response = JSONCurlUtil.get(${curl});
+
+		return ${response};
+	}
+
+	macro setupGlobalDocumentFolder {
+		Variables.assertDefined(parameterList = ${groupName});
+
+		var response = DocumentFolderAPI.addDocumentFolder(
+			externalReferenceCode = ${externalReferenceCode},
+			groupName = ${groupName},
+			name = ${name});
+
+		var documentFolderId = JSONPathUtil.getIdValue(response = ${response});
+		var externalReferenceCode = JSONPathUtil.getErcValue(response = ${response});
+		static var staticResponse1 = ${response};
```

Comment:
Any reason we set the static response here? What about setUpGlobalDocumentFolderIdAndErc{} in which pase the repsonse as a paramater?

Reply:
I think we can delete staticResponse1, and this method will return staticDoucmentFolderId and staticDoucmentFolderErc, how about this?